contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -1020,7 +1107,7 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
         }
       }
 
-      _validateDestFamilyAddress(chainFamilySelector, onRampTokenTransfers[i].destTokenAddress);
+      _validateDestFamilyAddress(chainFamilySelector, onRampTokenTransfers[i].destTokenAddress, 1);
```

Comment:
nit: Can we add a comment about how this hard coded gas limit is acceptable for Solana since this loop is already being called in the context of a token transfer, and therefore destTokenAddress does need to be checked for validity on SVM?